### DISCUSSION TITLE
Is it possible to host CompareFace without docker?

### DISCUSSION THREAD

**Discussion by ajjju** [exadel-inc/CompreFace#713] [General]
Hey, 

as we all know Docker is going to be paid from 31st January 2022,

So can you please provide the alternative for Docker where we can host compare face or is there any other way for free usage?

**Comment by Nexulo** [comment]
As I understand it, Docker is only chargeable for large enterprises (250+ employees). Or am I misunderstanding something?

**Reply by ajjju** [reply]
Yes, you are right but, i am going to use ComareFace using Docker with larger scale approximate thousand of employee.

That's why my concern is docker is free or not in this case and it would be better if we can use CompareFace without Docker.

**Reply by pospielov** [reply]
Only Docker Desktop will be paid, Docker Engine is open source and free.
The problem is that Docker Engine is built only for Linux. So if you use Linux, you can use Docker for free
You can start CompreFace without Docker, but we don't have instructions on how to do it and it could be very specific for every operating system. CompreFace consists of four servers and a database, so it's almost impossible to create instructions for all servers and all operating systems.
So, I recommend using Linux, so you can use docker for free.
